recipe/meta.yaml

```
@@ -10,35 +10,45 @@ source:
   sha256: bfa03788803061c80c005e28fab5d96f465fcf1c7d28a1eefb1631fb4a2ab67f
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [py<39]
 
 requirements:
   host:
-    - python >=3.7
+    - python
     - pip
     - setuptools
+    - wheel
   run:
-    - python >=3.7
+    - python
     - google-api-core >=1.34.1,<3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,!=2.5.*,!=2.6.*,!=2.7.*,!=2.8.*,!=2.9.*,!=2.10.*
     - google-auth >=2.14.1,<3.0.0dev,!=2.24.0,!=2.25.0
     - proto-plus >=1.22.3,<2.0.0dev
```

```suggestion
    - proto-plus >=1.22.3,<2.0.0dev
    - proto-plus >=1.25.0,<2.0.0dev # [py>=313]
```
https://github.com/googleapis/google-cloud-python/blob/google-ai-generativelanguage-v0.6.12/packages/google-ai-generativelanguage/setup.py